src/Compilers/Core/Portable/PublicAPI.Unshipped.txt

```diff
@@ -8,6 +8,7 @@ Microsoft.CodeAnalysis.GeneratorAttributeSyntaxContext.GeneratorAttributeSyntaxC
 Microsoft.CodeAnalysis.GeneratorAttributeSyntaxContext.SemanticModel.get -> Microsoft.CodeAnalysis.SemanticModel!
 Microsoft.CodeAnalysis.GeneratorAttributeSyntaxContext.TargetNode.get -> Microsoft.CodeAnalysis.SyntaxNode!
 Microsoft.CodeAnalysis.GeneratorAttributeSyntaxContext.TargetSymbol.get -> Microsoft.CodeAnalysis.ISymbol!
+Microsoft.CodeAnalysis.GeneratorDriver.ReplaceGenerators(System.Collections.Immutable.ImmutableArray<Microsoft.CodeAnalysis.ISourceGenerator!> generators) -> Microsoft.CodeAnalysis.GeneratorDriver!
```

Comment:
@333fred This will need an API review. @chsienki if you have specific concerns let's address them before the meeting. The "ReplaceGenerators" name matches the "ReplaceAdditionalTexts" that already exists on GeneratorDriver, and it behaves the same way.

Reply:
This was discussed and approved in https://github.com/dotnet/roslyn/issues/62796.